docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -17,6 +17,7 @@ When using the DMT, bear the following in mind:
 ** The source cluster must be running and populated with required data
 ** The application must be using the source cluster successfully before migration
 ** Before starting the migration, remember to shutdown any applications that are using the source cluster
+** If migrating Replicated map metadata, you must add the custom JAR to your class path and restart the cluster. If this is not done, the Replicated map metadata is not migrated. For further information on adding the custom JAR to your classpath, refer to the https://docs.oracle.com/javase/6/docs/technotes/tools/windows/classpath.html[Oracle documentation,window=_blank].
```

Comment:
We'll add an option to enable/disable replicated map metadata migration via a flag, will let you know when that happens